tests/utils.go

```diff
@@ -1435,3 +1437,25 @@ func GetNodeWithHugepages(virtClient kubecli.KubevirtClient, hugepages k8sv1.Res
 	}
 	return nil
 }
+
+// SkipIfVersionBelow will skip tests if it runs on an environment with k8s version below specified
+func SkipIfVersionBelow(virtClient kubecli.KubevirtClient, message string, version string) {
```

Comment:
You can fetch a new client. Should be cheap. We do that with other helper functions too, so that you don't have to permanently pass clients around..